### PR TITLE
Update distill lstm readme

### PR DESCRIPTION
**Pull request by LiuChiachi**
### PR types

Others
### PR changes

Docs
### Describe

Fix some dead links, and add some usage description for BERT fine-tuning on ChnSentiCorp.

### CODE REVIEW

File: examples/model_compression/distill_lstm/README.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ python -u ./run_glue.py \
     --n_gpu 1 \
 
 ```
+
+如果需要训练基于ChnSentiCorp数据集的BERT finetuning模型，可以将[glue目录](https://github.com/PaddlePaddle/PaddleNLP/tree/develop/examples/glue)下的`run_glue.py`稍作修改，在字典`TASK_CLASSES`中配置ChnSentiCorp数据集相关的信息，增加一组键值对：`"senta":(ChnSentiCorp, Accuracy)`，并且记得导入数据集`from paddlenlp.datasets import ChnSentiCorp`。训练时将参数`TASK_NAME`设为`"senta"`，参数`model_name_or_path`可以是`bert-wwm-ext-chinese`或者`bert-base-chinese`。
```
Comment (ZeyuChen) [inline examples/model_compression/distill_lstm/README.md]: 是否可以参考
https://github.com/PaddlePaddle/PaddleNLP/tree/develop/examples/text_classification/pretrained_models

引导用户参考这个链接的代码是否更为简单点？感觉你这个指引太复杂了

**Review by ZeyuChen — COMMENTED** [review]
是否进行跳转到文本分类预训练模型模型那里更为合适？

File: examples/model_compression/distill_lstm/README.md (hunk 1 above)
Comment (LiuChiachi) [reply examples/model_compression/distill_lstm/README.md]: 如果参考senta的这个链接的话，用户也需要把代码里的预训练模型从Ernie改成BERT。不过我之前这个描述确实有点啰嗦，可以简化成一句话来描述。那是指引用户去文本分类项目里换预训练模型简单，还是在run_glue里增加支持的数据集简单呢

Comment (ZeyuChen) [reply examples/model_compression/distill_lstm/README.md]: 指引到文本分类那里就好，文本分类那里也可以选bert模型，开发者在那边的文档里应该能理解。你这边可以简化

File: examples/model_compression/distill_lstm/README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ pip install paddlenlp\>=2.0rc
 
 对于BERT的fine-tuning任务，本实验中使用了预训练模型`bert-bas-uncased`、`bert-wwm-ext-chinese`、`bert-base-chinese`。同样，这几个模型在训练时会被自动下载到`paddlenlp.utils.env.DATA_HOME`路径下。例如，对于`bert-base-uncased`模型，在linux系统下，会被下载到`~/.paddlenlp/models/bert-base-uncased`下。
 
-在中文数据集上的小模型训练的输入利用jieba分词，其中词表同本repo下[`example/text_classification/rnn`](https://github.com/PaddlePaddle/models/tree/develop/PaddleNLP/examples/text_classification/rnn)的词表，可通过运行以下命令进行下载：
+在中文数据集上的小模型训练的输入利用jieba分词，其中词表同本repo下[`example/text_classification/rnn`](https://github.com/PaddlePaddle/PaddleNLP/tree/develop/examples/text_classification/rnn)的词表，可通过运行以下命令进行下载：
```
Comment (ZeyuChen) [inline examples/model_compression/distill_lstm/README.md]: 链接要使用相对链接，不要用绝对链接，这样会导致develop发版后出错

**Review by ZeyuChen — COMMENTED** [review]
PaddleNLP内的链接改为相对链接

File: examples/model_compression/distill_lstm/README.md (hunk 1 above)
Comment (LiuChiachi) [reply examples/model_compression/distill_lstm/README.md]: 好的，感谢，已经修改

File: examples/model_compression/distill_lstm/README.md (hunk 2 above)
Comment (LiuChiachi) [reply examples/model_compression/distill_lstm/README.md]: 感谢建议，现已修改

**Review by ZeyuChen — APPROVED** [review]
LGTM
